sync-todo/v2/client/cpp/controllers/home_controller.cpp

```diff
@@ -90,7 +91,7 @@ HomeController::HomeController(AppState *appState): Controller(ftxui::Container:
     ftxui::Elements tasks;
     // If the user has toggled the checkbox to hide completed tasks, show only the incomplete task list.
     // Otherwise, show all items.
-    for (auto &item: itemList) {
+    for (auto const item: itemList) {
```

Comment:
Why this change?

Reply:
When I updated to v1.1.1, `auto &item` no longer works - I get this error:

`Non-const lvalue reference to type 'managed<...>' cannot bind to a temporary of type 'managed<...>'`

So I thought this was the correct fix, and it seems to work, but I'll defer to your expertise if there is a better/different way I should resolve this build error.

Reply:
Try `auto const &item`

Reply:
All good with `auto const &item`. Thanks!